tools/validate/wamr/validation.json

```diff
@@ -2642,6 +2642,16 @@
             }
         ]
     },
+    {
+        "module": "infc_assign_obj",
```

Comment:
There should already be a module named "infc_assign_obj", had better add the entry to that already existed module

Reply:
done